09-mapping.Rmd

```diff
@@ -44,26 +44,26 @@ nz_elev = rast(system.file("raster/nz_elev.tif", package = "spDataLarge"))
 ## Introduction
 
 A satisfying and important aspect of geographic research is communicating the results.
-Map making\index{map making} --- the art of cartography --- is an ancient skill involving communication, attention to detail, and an element of creativity.
+map-making\index{map-making} --- the art of cartography --- is an ancient skill involving communication, attention to detail, and an element of creativity.
 Static mapping in R is straightforward with the `plot()` function, as we saw in Section \@ref(basic-map).
 It is possible to create advanced maps using base R methods [@murrell_r_2016].
 The focus of this chapter, however, is cartography with dedicated map-making packages.
 When learning a new skill, it makes sense to gain depth-of-knowledge in one area before branching out.
-Map making is no exception, hence this chapter's coverage of one package (**tmap**) in depth rather than many superficially.
+map-making is no exception, hence this chapter's coverage of one package (**tmap**) in depth rather than many superficially.
```

Comment:
```suggestion
Map-making is no exception, hence this chapter's coverage of one package (**tmap**) in depth rather than many superficially.
```

Capital letter is missing